src/test/java/be/seeseemelk/mockbukkit/inventory/InventoryMockTest.java

```diff
@@ -573,4 +573,14 @@ void testAddMultipleViewersVarargsWithNullEntries()
 		assertThrows(NullPointerException.class, () -> inventory.addViewers(player1, player2, null, player3));
 	}
 
+	@Test
+	void testOpenInventoryAddViewers()
+	{
+		Player player = server.addPlayer();
+		player.openInventory(inventory);
+
+		assertEquals(1, inventory.getViewers().size());
+		assertTrue(inventory.getViewers().contains(player));
+	}
+
```

Comment:
One more test should be added to ensure they get removed from the viewers list when calling `#closeInventory`